javacutil/src/main/java/org/checkerframework/javacutil/TreeUtils.java

```diff
@@ -285,6 +286,20 @@ public static boolean isSelfAccess(ExpressionTree tree) {
     // This section of the file groups methods by their receiver type; that is, it puts all
     // `elementFrom*(FooTree)` methods together.
 
+    /**
+     * Return the type element corresponding to the given package declaration
+     *
+     * @param tree package declaration
+     * @return the element for the given package
+     */
+    public static TypeElement elementFromDeclaration(PackageTree tree) {
```

Comment:
This method doesn't seem to be used. It would fail, because for a `PackageTree` you should get a [PackageElement](https://github.com/openjdk/jdk/blob/master/src/java.compiler/share/classes/javax/lang/model/element/PackageElement.java), so the cast would fail.